src/Faker/Provider/pl_PL/LicensePlate.php

```diff
@@ -0,0 +1,571 @@
+            'kępiński' => ['KE'],
+            'kolski' => ['KL'],
+            'koniński' => ['KN'],
+            'kościański' => ['KS'],
+            'krotoszyński' => ['KR'],
+            'leszczyński' => ['LE'],
+            'międzychodzki' => ['MI'],
+            'nowotomyski' => ['NT'],
+            'obornicki' => ['OB'],
+            'ostrowski' => ['OS'],
+            'ostrzeszowski' => ['OT'],
+            'pilski' => ['P'],
+            'pleszewski' => ['PL'],
+            'poznański' => ['OZ', 'Z'],
+            'rawicki' => ['RA'],
+            'słupecki' => ['SL'],
+            'szamotulski' => ['SZ'],
+            'średzki' => ['SR'],
+            'śremski' => ['SE'],
+            'turecki' => ['TU'],
+            'wągrowiecki' => ['WA'],
+            'wolsztyński' => ['WL'],
+            'wrzesiński' => ['WR'],
+            'złotowski' => ['ZL'],
+        ],
+        'Z' => [
+            'Koszalin' => ['K'],
+            'Szczecin' => ['S', 'Z'],
+            'Świnoujście' => ['SW'],
+            'białogardzki' => ['BI'],
+            'choszczeński' => ['CH'],
+            'drawski' => ['DR'],
+            'goleniowski' => ['GL'],
+            'gryficki' => ['GY'],
+            'gryfiński' => ['GR'],
+            'kamieński' => ['KA'],
+            'kołobrzeski' => ['KL'],
+            'koszaliński' => ['KO'],
+            'łobeski' => ['LO'],
+            'myśliborski' => ['MY'],
+            'policki' => ['PL'],
+            'pyrzycki' => ['PY'],
+            'sławieński' => ['SL'],
+            'stargardzki' => ['ST'],
+            'szczecinecki' => ['SZ'],
+            'świdwiński' => ['SD'],
+            'wałecki' => ['WA'],
+        ],
+        'U' => [
+            'Siły Zbrojne Rzeczypospolitej Polskiej' => ['A', 'B', 'C', 'D', 'E', 'G', 'I', 'J', 'K', 'L'],
+        ],
+        'H' => [
+            'Centralne Biuro Antykorupcyjne' => ['A'],
+            'Służba Ochrony Państwa' => ['BA', 'BB', 'BE', 'BF', 'BG'],
+            'Służba Celno-Skarbowa' => ['CA', 'CB', 'CC', 'CD', 'CE', 'CF', 'CG', 'CH', 'CJ', 'CK', 'CL', 'CM', 'CN', 'CO', 'CP', 'CR'],
+            'Agencja Bezpieczeństwa Wewnętrznego' => ['K'],
+            'Agencja Wywiadu' => ['K'],
+            'Służba Kontrwywiadu Wojskowego' => ['M'],
+            'Służba Wywiadu Wojskowego' => ['M'],
+            'Policja' => ['PA', 'PB', 'PC', 'PD', 'PE', 'PF', 'PG', 'PH', 'PJ', 'PK', 'PL', 'PL', 'PL', 'PL', 'PL', 'PM', 'PN', 'PP', 'PS', 'PT', 'PU', 'PW', 'PZ'],
+            'Straż Graniczna' => ['WA', 'WK'],
+        ],
+    ];
+
+    /**
+     * Generates random license plate.
+     * Generation method based on the description from https://pl.wikipedia.org/wiki/Tablice_rejestracyjne_w_Polsce#Tablice_standardowe
+     *
+     * @param  array|null $voivodeships
+     * @param  array|null $counties
+     *
+     * @return string
+     */
+    public function licensePlate($voivodeships = null, $counties = null)
+    {
+        if (is_array($voivodeships)) {
```

Comment:
Since we are on PHP >= 7.1, you can use null coalescence operator (`??`) - for example: `array_filter($voivodeships ?? [])`